cmd/action/mux.go

```diff
@@ -43,8 +43,9 @@ func NewMux() *Mux {
 }
 
 var (
-	defaultMux = NewMux()
-	inMemory   = regexp.MustCompile("^file:.*:memory:$|:memory:|^file:.*mode=memory.*")
+	// DefaultMux is the default Mux that is used by the different commands.
+	DefaultMux = NewMux()
+	reMemMode  = regexp.MustCompile("^file:.*:memory:$|:memory:|^file:.*mode=memory.*")
```

Comment:
```suggestion
	reMemMode  = regexp.MustCompile(":memory:|^file:.*mode=memory.*")
```

That does the same, doesn't it?

Reply:
Yes, it was there. I didn't write it.